js/contextMenus.js

```diff
@@ -197,6 +197,25 @@ function moreBookmarksTemplateInit (allBookmarkItems, bookmarks, activeFrame) {
   return template
 }
 
+function usernameTemplateInit (usernames, origin, action) {
+  let items = []
+  for (var username in usernames) {
+    let password = usernames[username]
+    items.push({
+      label: username,
+      click: (item, focusedWindow) => {
+        console.log('item clicked', username, password)
```

Comment:
Yep, but this should be good to go by end of day. I'll open separate pull req for the other follow ups.